src/main/java/overclocked/stl/io/OverclockedController.java

```diff
@@ -0,0 +1,30 @@
+
+package overclocked.stl.io;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+
+/** 
+ * An extension of the {@link GenericHID} class that adds some useful utilities.
```

Comment:
There’s new utility, so maybe it should be singular (at least for now)?

Reply:
The plural is fine, saves a change later and doesn't really cause any confusion. "Adds a useful utility" sounds exceedingly vague and "Adds a utility that makes creating JoystickButtons cleaner" _is_ exceedingly verbose.